chain-impl-mockchain/src/certificate/vote_plan.rs

```diff
@@ -83,6 +83,10 @@ impl VoteOptions {
             num_choices: num_choices & Self::NUM_CHOICES_MAX,
         }
     }
+
+    pub fn choices(&self) -> u8 {
```

Comment:
`into_byte()` ?

Reply:
this is going to be extended (or not) in the future, so I think it would be better to be straight forward about it

Reply:
I'll use `as_byte` since `into` methods are usually supposed to consume the value.